ssl/ssl_locl.h

```diff
@@ -2263,7 +2265,7 @@ __owur int tls13_update_key(SSL *s, int send);
 __owur int tls13_hkdf_expand(SSL *s, const EVP_MD *md,
                              const unsigned char *secret,
                              const unsigned char *label, size_t labellen,
-                             const unsigned char *hash,
+                             const unsigned char *data, size_t datalen,
```

Comment:
It's because, starting with this PR, we pass something other than a hash in for the first time, i.e. we pass the ticket_nonce.